service/frontend/workflow_handler_test.go

```diff
@@ -1428,23 +1428,23 @@ func (s *workflowHandlerSuite) TestGetHistory() {
 	namespaceID := namespace.ID(uuid.New())
 	namespaceName := namespace.Name("test-namespace")
 	firstEventID := int64(100)
-	nextEventID := int64(102)
+	nextEventID := int64(103)
```

Comment:
Can we rename this to lastEventID? It seems like this is not the next event after the first event.

Reply:
I don't have any prior knowledge around these tests and do not know what the intentions are to name them that way in the first place. There are plenty other places where this choice of word prefix is used and I'd rather keep it standardized (at least shouldn't deviate for the purpose of this PR)